### PR TITLE
Releasing v2023.5

### PR DESCRIPTION
**Pull request by JulienPalard**

### CODE REVIEW

**Review by hugovk — APPROVED** [review]
Perhaps we could include https://github.com/python/python-docs-theme/pull/121 as well? I've appproved it but not merged it.

**Comment by pradyunsg** [comment]
> Perhaps we could include #121 as well?

+! to this.

**Comment by JulienPalard** [comment]
rebased on top of #121

**Review by AA-Turner — APPROVED** [review]
Thanks Julien!

A

**Comment by AA-Turner** [comment]
(#99 needs fixes, which I have found by updating https://github.com/python/cpython/pull/99381 -- working on a PR now. Sorry!)

**Comment by hugovk** [comment]
https://github.com/python/python-docs-theme/pull/127 merged 🚀

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
 =========
 
+`2023.4 <https://github.com/python/python-docs-theme/releases/tag/v2023.4>`_
+----------------------------------------------------------------------------
+
+- Add a dark theme. (#44)
+  Contributed by Nils K
+- Fix: Remove searchbox id from form. (fixes #117)
+  Contributed by Nils K
+- Update ``python-docs-theme`` to work with Sphinx 5 & 6. (#99)
```
Comment (AA-Turner) [inline CHANGELOG.rst:12]:
```suggestion
- Update ``python-docs-theme`` to work with Sphinx 5 & 6. (#99 & #127)
```

**Review by AA-Turner — APPROVED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+  Contributed by Adam Turner
+- Override font for .sig for consistency with other code blocks. (#121)
```
Comment (ezio-melotti) [inline CHANGELOG.rst:14]:
```suggestion
- Override font for ``.sig`` for consistency with other code blocks. (#121)
```
Minor nit.

**Comment by JulienPalard** [comment]
Rebased on top of #127.

**Comment by hugovk** [comment]
Would be great to include https://github.com/python/python-docs-theme/pull/128 as well if someone has a chance to review it :) If not, it can wait until next time.

**Comment by JulienPalard** [comment]
I'll have to disconnect in a few hours for like one week (holidays) so I'm OK for waiting for #128 and release next week with more time to handle it (rolling it to production, monitoring feedbacks,…).

**Comment by hugovk** [comment]
Sounds good to me, enjoy your holiday!

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+
+- Add a dark theme. (#44)
+  Contributed by Nils K
+- Fix: Remove searchbox id from form. (fixes #117)
+  Contributed by Nils K
+- Update ``python-docs-theme`` to work with Sphinx 5 & 6. (#99 & #127)
+  Contributed by Adam Turner
+- Override font for ``.sig`` for consistency with other code blocks. (#121)
+  Contributed by Chris Warrick
+
```
Comment (hugovk) [inline CHANGELOG.rst:16]: #128 has been merged:

```suggestion
- Dark mode: add class to invert image brightness. (#128)
  Contributed by Hugo van Kemenade

```

**Comment by JulienPalard** [comment]
I'm back! Does this release looks OK to you?

**Comment by hugovk** [comment]
Welcome back, hope you had a relaxing time! The release looks good!

**Comment by AA-Turner** [comment]
Same here, looks good!

A

**Comment by JulienPalard** [comment]
Released!

**Pull request merged**